mlir/test/lit.cfg.py

```diff
@@ -45,10 +45,10 @@
                    'lit.cfg.py', 'lit.site.cfg.py']
 
 # test_source_root: The root path where tests are located.
-config.test_source_root = os.path.dirname(__file__)
+#config.test_source_root = os.path.dirname(__file__)
 
 # test_exec_root: The root path where tests should be run.
-config.test_exec_root = os.path.join(config.mlir_obj_root, 'test')
+#config.test_exec_root = os.path.join(config.mlir_obj_root, 'test')
```

Comment:
these 2 changes break default MLIR setup. is there a way to get around this?

Reply:
I don't think they break anything. They're redundant with line 27 and 30.

Reply:
therefore is it possible to leave `lit.cfg.py` as-is?

Reply:
Yes. I've revoked the changes